charts/kashti/values.yaml

```diff
@@ -18,9 +18,11 @@ service:
   internalPort: 80
 ingress:
   enabled: false
+  # From Kubernetes 1.18+ this field is supported in case your ingress controller supports it. When set, you do not need to add the ingress class as annotation.
+  ingressClassName:
   ## Used to create Ingress record (should used with service.type: ClusterIP).
   hosts:
-  - chart-example.local
+    - chart-example.local
```

Comment:
Nit: Both of these work and we've tended to favor doing it _without_ the extra space. I'm sure not everyone agrees with that (the same way some people clash over tabs v spaces), but since this change isn't really needed for the PR, let's undo this one.

Reply:
Sorry, autoformat of my IDE. Reverted.